### ISSUE TITLE
The attribute aria-expanded is not supported by the role menuitem.

### ISSUE THREAD

**Issue by jlurena** [dequelabs/axe-core-gems#76]
Please see the same issue I posted here https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/630

A menu item, acting as a submenu given a role of `menuitem` should have an `aria-expanded` attribute. I am being thrown an error when I do this.

Something like this raises an error:
```
<ul role="menubar">
  <li>
    <a role="menuitem" aria-expanded="false" href="someurl.com">Some URL</a>
      <ul role="menu">
        <li><a href="https://google.com">Google.com</a></li>
        <li><a href="https://github.com">Github.com</a></li>
      </ul>
  </li>
</ul>
```

> The attribute aria-expanded is not supported by the role menuitem.

According to https://github.com/w3c/aria/issues/454 this is valid aria spec.

This is also shown on an example on W2 
https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html

**Comment by jkodu** [comment]
@jlurena thanks for reporting this. I will investigate and get back to you.

**Comment by jlurena** [comment]
Specific aria-spec: https://www.w3.org/TR/wai-aria-practices-1.1/#menu

**Comment by jlurena** [comment]
Was this fixed?

**Comment by jeeyyy** [comment]
@jlurena 

Thanks for this issue. It looks like [`js-ally-plugin` does not report this as an issue now](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/630#issuecomment-647056905).

The latest version of [axe-core v4.0.2](https://github.com/dequelabs/axe-core) does not throw an issue for the above code snippet either. 

I would recommend moving to the latest version of `axe-core-*-gems`, which uses the latest of `axe-core`. We have re-engineered the way the gems/ packages can be consumed.

**Comment by jeeyyy** [comment]
@jlurena feel free to re-open the issue, if you need further input.
